content/courses/tokens-and-nfts/nfts-with-metaplex-core.md

```diff
@@ -0,0 +1,834 @@
+---
+title: Create Solana NFTs With Metaplex Core
+objectives:
+  - Explain NFTs and how they're represented on the Solana network
+  - Explain Collections and how do they differ between the Core Program and the Token Metadata Program
+  - Explain the role of the Metaplex Core program
+  - Create and update a Collection using the Metaplex JS SDK
+  - Create and update Assets using the Metaplex JS SDK
+description:
+  "How to create NFTs in TypeScript with Metaplex Core program and Irys
+  permanent storage service."
+tags:
+  - quickstart
+  - metaplex
+  - typescript
+keywords:
+- typescript
+- npm
+- yarn
+- package
+- tutorial
+- digital assets
+- intro to solana nfts
+- blockchain developer
+- blockchain tutorial
+- web3 developer
+---
+
+### Summary
+
+- **Non-Fungible Tokens (NFTs)** are Digital Assets with 0 decimals, and 
+  a maximum supply of 1
+- **Metadata** attaches additional properties to Assets and Collection. 
+  Metadata includes the token name and a link to an offchain JSON file. 
+  This JSON file contains links to artwork and other media files, any 
+  special traits the NFT has, and more.
+- The [Metaplex Core Program](https://developers.metaplex.com/core) is an 
+  onchain program that sheds the complexity and technical debt of previous 
+  standards and provides a clean and simple core spec for digital assets 
+  using a single account design
```

Comment:
I feel like this summary is misleading at the start due to saying what an nft is and then stating actually we are not doing that.

Should probably be an introduction to Core and then have a differences area which shows why Core has an advantage over Token Metadata.